build.gradle

```diff
@@ -19,6 +19,7 @@ buildscript {
     dependencies {
         classpath group: 'com.google.protobuf', name:'protobuf-gradle-plugin', version: protobufGradlePlugin
         classpath "gradle.plugin.org.nosphere.apache:creadur-rat-gradle:0.3.0"
+        classpath 'org.kordamp:markdown-gradle-plugin:1.1.0'
```

Comment:
Is there a way of making this more self-contained? I don't like very much the idea of adding this dependency globally like this. The same comment applies the `allProjects` block below. Ideally, we have some sort of `genHtmlDocs` and make it self-contained.

Reply:
Unfortunately, this doesn't seem to be possible, there is an only single block of build script {}, and these need to be added to that block. 
I have a condition in `mkdocs.gradle` file to apply the plugins only if it is root project.